src/main/drivers/usb_msc_f4xx.c

```diff
@@ -173,4 +171,13 @@ void systemResetToMsc(int timezoneOffsetMinutes)
 #endif
     NVIC_SystemReset();
 }
+
+void systemResetFromMsc(void)
+{
+    *((uint32_t *)0x2001FFF0) = 0xFFFFFFFF;
```

Comment:
Do we want to introduce a define for this address?

Reply:
Truthfully I'm not sure. It's only used in this file. I'm wondering why we don't shift to using the persistent layer rather than the hardcoded vector - but I don't have any experience with the low level hardware. @jflyper any thoughts?

Reply:
> why we don't shift to using the persistent layer rather than the hardcoded vector

Just overlooked the potential use case 😓 

Theoretically, in F4 case, the system is in known soft reset state when `init` is called (`checkForBootLoaderRequest` ensures a soft reset was issued by ourself), so it is safe to assume the end of the RAM region (`0x2001FFF0`) persist across the soft reset, although the particular address location itself is not protected well (linker-wise).

That being said, I think the MSC activation state should be converted to use the persistent object layer, so that there is no need of above knowledge when looking at the MSC activation state code.